### DISCUSSION TITLE
MongoDB connector

### DISCUSSION THREAD

**Discussion by talelk** [ConduitIO/conduit#898] [Connectors]
idealy, ,need a mongodb connector

**Comment by hariso** [comment]
@talelk Thanks for reaching out! We're working on a Conduit [source](https://github.com/conduitio-labs/conduit-connector-mongo/pull/29) and [destination](https://github.com/conduitio-labs/conduit-connector-mongo/pull/30) for MongoDB.

**Comment by raulb** [comment]
@talelk Thanks for bringing this up. Closing this loop here as it shipped already https://github.com/conduitio-labs/conduit-connector-mongo.
